examples/slu/slurp/requirements.txt

```diff
@@ -0,0 +1,6 @@
+progress==1.5
```

Comment:
Dont put requirements in slu dir, put them in requirements dir. And add it to the "all" call inside setup.py